docs/introduction/faq-and-common-issues.md

```diff
@@ -0,0 +1,74 @@
+# FAQ and Common Issues
+
+This section provides only the basic answers to some of the most frequently asked questions.
+For more detailed information about the Shopsys Framework, please see [Shopsys Framework Knowledge Base](../index.md).
+
+## What are the phing targets?
+Every phing target is a task that can be executed simply by php phing <target-name> command.
+See more about phing targets in [Console Commands for Application Management (Phing Targets)](./console-commands-for-application-management-phing-targets.md).
+
+## What are the data fixtures good for?
+Data fixtures are actually demo data available in the Shopys Framework.
+There are two kinds of demo data, the singledomain and the multidomain.
+For the installation of the basic demo data, you need to have just one domain configured, use the phing target `db-fixtures-demo-singledomain`.
+For the installation of the multidomain demo data, you need to have just at least two domains configured, use the phing target `db-fixtures-demo-multidomain`.
+These phing targets are also triggered as the part of others phing targets, see `build.xml` and `build-dev.xml`.
+Demo data are used for automatic tests and also for installation of demo shop with prepared data.
+
+## How to change a domain URL?
+The change of domain url requires two steps.
+In the first step, you need to modify the domain url in the configuration file `app/config/domains_urls.yml`.
```

Comment:
It is out of the scope of this US, but `domain.yml` is also in https://github.com/shopsys/shopsys/blob/master/docs/introduction/how-to-set-up-domains-and-locales.md#4-change-the-url-address-for-an-existing-domain

Please correct it also there OR create an issue that would fix it